tests/chainer_tests/links_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -243,4 +244,23 @@ def test_backward_gpu(self):
         gy = cuda.to_gpu(self.gy)
         self.check_backward(x, gy)
 
+
+class TestInvalidInput(unittest.TestCase):
+
+    def setUp(self):
+        self.link = links.BatchNormalization(3)
+
+    def test_invalid_initialize(self):
```

Comment:
This test fixture should be separated as another class because it does not depend on `setUp` of this class.